packages/tsup-configs/README.md

```diff
@@ -51,6 +73,28 @@ import { defineBuildConfig } from "@workleap/tsup-configs";
 export default defineBuildConfig();
 ```
 
+You can override any existing options:
+
+```ts
+// tsup.build.ts
+import { defineBuildConfig } from "@workleap/tsup-configs";
+
+export default defineBuildConfig({
+    clean: true
+});
+```
+
+The provided options will be merged with the default options. Given that a provided option match a default option, it will override the default option. If you prefer to extend the default option, you can import the `DefaultBuildOptions` object and handle the merging code in your configuration file:
```

Comment:
```suggestion
The provided options will be merged with the default options. Given that a provided option matches a default option, it will override the default option. If you prefer to extend the default option, you can import the `DefaultBuildOptions` object and handle the merging code in your configuration file:
```